### ISSUE TITLE
Sidebar broken with the new Firefox update

### ISSUE THREAD

**Issue by LinkIT3** [Naezr/ShyFox#207]
After updating Firefox to version 133.0, which introduced the built-in sidebar, a few problems have appeared.

- I can't remove the built-in sidebar

![Screenshot 2024-11-27 130431](https://github.com/user-attachments/assets/c61a17c9-ab28-4abe-97ad-5a95b6e276e1)

- The window is stuck like when the ShyFox sidebar is open. 

![Screenshot 2024-11-27 130506](https://github.com/user-attachments/assets/edf7c6cf-94b5-432c-a9b8-834d7e607ed1)

**Comment by Sneaky1000** [comment]
Yeah, the bookmarks toolbar is stuck for me. It looks like it's getting pushed under the main window itself.

**Comment by LinkIT3** [comment]
I resolved by reinstalling Firefox

**Comment by simonk83** [comment]
I can't get the sidebar to hide at all, not sure if that's related. Weirdly it does hide when enabled clean mode, but not just hiding the sidebar itself
